ipatests/test_integration/test_subids.py

```diff
@@ -268,3 +269,31 @@ def test_sudid_match_shows_uid(self):
                                           f"--subuid={subuid}"])
         owner = self._parse_result(result)["owner"]
         assert owner == uid
+
+    def test_nsswitch_doesnot_contain_subid_entry(self):
+        """
+        This testcase checks that when ipa-client-install
+        is installed without subid option, the nsswitch.conf
+        doesnot contain subid entry
+        """
+        cmd = self.clients[0].run_command(
+            ["grep", "^subid", "/etc/nsswitch.conf"]
+        )
+        assert cmd.returncode == 1
+        assert 'subid' not in /etc/nsswitch.conf
```

Comment:
@menonsudhir 
please run make fastlint before you push updates to this PR, it will detect this type of issue. This line is not understood by python.